tensorflow/contrib/cmake/external/eigen.cmake

```diff
@@ -7,7 +7,7 @@
 
 include (ExternalProject)
 
-set(eigen_archive_hash "802d984ade26")
+set(eigen_archive_hash "bbffe8ed5f26")
```

Comment:
Maybe this new bazel option can help: http://www.bazel.io/docs/be/workspace.html#new_http_archive.strip_prefix